<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multi3dviewer</class>
 <widget class="QDialog" name="multi3dviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Cam 3D Viewer</string>
  </property>
  <widget class="glwidgetcloud" name="viewer" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>831</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="viewCam1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>626</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cam 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="viewCam2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>654</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cam 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="doRemap">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mix Cams With  Cal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keppCloud">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>635</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep Cloud</string>
   </property>
  </widget>
  <widget class="QPushButton" name="videoRecBut">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>636</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Video Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="videoPlay">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>636</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Play Video</string>
   </property>
  </widget>
  <widget class="QSlider" name="videoSlider">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>670</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="videoControl">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>666</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>604</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Active?</string>
   </property>
  </widget>
  <widget class="QSlider" name="maxDist1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>629</y>
     <width>251</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>7000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>7000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="maxDist2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>654</y>
     <width>251</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>7000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>7000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>603</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Distance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fiSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>694</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>610</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Video Size : </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxSizeMB">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>606</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>610</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>MB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>698</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Video Size : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>698</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>MB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Background :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="colR">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>715</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>720</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>720</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="colG">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>715</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>720</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="colB">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>715</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="pSkip">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>680</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>683</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pt. Skip</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glwidgetcloud</class>
   <extends>QWidget</extends>
   <header>glwidgetcloud.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
